### PR TITLE
poc: first integration compose setup

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: integration/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-stas: &common-opi
+    image: ghcr.io/opiproject/opi-poc:main
```
Comment (mestery) [inline integration/docker-compose.yml:4]: Have we pushed a container to GHCR yet?

File: integration/host-cpu/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM fedora:33
```
[the remark this answers is not in the document]
Comment (mestery) [reply integration/host-cpu/Dockerfile:1]: Agree, lets make sure these are versions which will get security updates.

File: integration/xpu-cpu/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+FROM fedora:33
```
Comment (mestery) [inline integration/xpu-cpu/Dockerfile:1]: Same here.

**Comment by glimchb** [comment]
1. we don't have top publish to ghcr , in this case everything is built from dockerfiles
2. i specifically want fedora 33 here because in the next commit i want to try dependabot on dockerfiles and see if it detects and updates our dockerfiles automatically

File: integration/docker-compose.yml (hunk 1 above)
Comment (glimchb) [reply integration/docker-compose.yml:4]: no, and we don;t have to, compose will build everything from dockerfiles

File: integration/host-cpu/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply integration/host-cpu/Dockerfile:1]: yep, wanna test auto-update with dependabot

File: integration/host-cpu/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+RUN useradd -rm -d /home/test -s /bin/bash -g root -G ssh_keys -u 1000 test 
+RUN  echo 'test:test' | chpasswd
+RUN sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+RUN ssh-keygen -A
+
+EXPOSE 22
+CMD ["/usr/sbin/sshd", "-D"]
+
+# TODO: docker pull linuxserver/openssh-server
```
[the remark this answers is not in the document]
Comment (seroyer) [reply integration/host-cpu/Dockerfile:15]: I am not a fan of cryptic TODOs in the code.  In the current context, this TODO doesn't make sense.

**Pull request merged**